Security/src/EOMT.ps1

```diff
@@ -790,7 +790,8 @@ try {
         #Normal run
         $IsVulnerable = Get-ServerVulnStatus
         if ($IsVulnerable) {
-            $Message = "$env:computername is vulnerable: applying mitigation"
+            $Message = "$env:computername is vulnerable: needs to apply mitigation"
+            $Message + "`n" + "Please use Microsoft Update to install the latest Security Update For Exchange Server (KB5000871)"
```

Comment:
From the design aren't we giving individual links to the SU version based on the customer's Exchange CU version?